.travis.yml

```diff
@@ -1,6 +1,24 @@
+sudo: false
+
 language: groovy
+
 jdk:
-- oraclejdk8
+  - oraclejdk7
+  - oraclejdk8
+
+env:
+  - GRADLE_VERSION=1.12
+  - GRADLE_VERSION=2.4
+  - GRADLE_VERSION=2.5
+
+before_script:
+  - cp -rf gradle/wrapper/gradle-wrapper.template gradle/wrapper/gradle-wrapper.properties
```

Comment:
AAA you would like to append the last line. Maybe you're right. This would
be less invasive.
On Jul 11, 2015 10:28, "Marcin Grzejszczak" <marcin@grzejszczak.pl> wrote:

> I won't be able to run gradlew locally then, won't I?
>